ci/Dockerfile

```diff
@@ -1,21 +1,7 @@
-FROM php:8.3
+FROM ubuntu:latest as base
```

Comment:
Also you do not need `as base`...  This is only needed when you are doing complex multi stage Docker builds: https://docs.docker.com/build/building/multi-stage/#name-your-build-stages

Which you aren't doing here.